src/sentry/conf/server.py

```diff
@@ -378,6 +378,8 @@
 CELERY_DEFAULT_EXCHANGE_TYPE = "direct"
 CELERY_DEFAULT_ROUTING_KEY = "default"
 CELERY_CREATE_MISSING_QUEUES = True
+CELERY_REDIRECT_STDOUTS = False
```

Comment:
As a sidenote, I plan to put in a HumanRenderer after we get the machine stuff ready so that we aren't just printing `event=(What we actually want to see)` and make it a lot less noisy.